src/style.css

```diff
@@ -0,0 +1,132 @@
+body {
```

Comment:
- [ ] Your project should look like this template:
    ![imagen](https://user-images.githubusercontent.com/60789317/140101584-b0275c54-a40c-4142-8736-c6dce9580a79.png)
    - Your project looks like this:
    ![imagen](https://user-images.githubusercontent.com/60789317/140101715-85b92366-e2bb-4bbd-a88b-c8cacc5fe604.png)

Reply:
This issue is well resolved! ✔️